src/xdocs/index.xml.vm

```diff
@@ -217,6 +217,19 @@
       </div>
     </section>
 
+    <section name="Supported Java Language Version">
+      <p>
+        Checkstyle's grammar and checks currently support all Java language features introduced
```

Comment:
Let's make this only about what syntax we can parse, Check support will likely never be comprehensive.

Reply:
done